apps/desktop/desktop/src/main/java/bisq/desktop/components/controls/validator/NumberValidator.java

```diff
@@ -152,6 +115,10 @@ protected void eval() {
         }
     }
 
+    private String cleanUpFormatting(String inputText) {
+        return inputText.replace(",", "");
```

Comment:
Unfortunately diff. locales use diff. separator here. I am not a fan of those local number formats and we could just use one default number format independent of users locale.

Reply:
I think this would be the safest as well. What do you think about using `Locale.US` as default?
I think ideally we only have the number formatted when the input is in read mode, and then remove the formats when writing, but that adds a lot of unnecessary complexity imo.